pkg/net/libp2p/channel.go

```diff
@@ -326,7 +325,7 @@ func (c *channel) SetFilter(filter net.BroadcastChannelFilter) error {
 	c.pubsubMutex.Lock()
 	defer c.pubsubMutex.Unlock()
 
-	c.pubsub.UnregisterTopicValidator(c.name)
+	_ = c.pubsub.UnregisterTopicValidator(c.name)
```

Comment:
Let's log this error. Agreed, we should never have this problem but better be safe than sorry.